TCL4 Data Management/utm-tcl4-dmp-daa.yaml

```diff
@@ -424,8 +451,14 @@ definitions:
       transmissionDelayV2v:
         description: >-
           Transmission time delay associated with transmitting data packet
-          information from a source to a destination (V2V). Include 3 decimal
-          places of precision. (sec)
+          information from a source to a destination (V2V).
+
+          Please report in milliseconds (ms).
+
+          NOTE:
+            DMP-Rev3 asks asks this to be reported with 3 decimals of precision.
```

Comment:
has ask repeated twice in the first sentence (459).

Reply:
Good catch.